composer.json

```diff
@@ -47,12 +47,14 @@
         "laminas/laminas-cli": "^1.7",
         "laminas/laminas-coding-standard": "~2.5.0",
         "laminas/laminas-config-aggregator": "^1.13",
-        "laminas/laminas-serializer": "^2.14",
+        "laminas/laminas-serializer": "3.0.x-dev",
+        "laminas/laminas-servicemanager": "^4.0.0-rc3",
```

Comment:
require-dev seems not needed for `laminas/laminas-servicemanager` as it already in `require` above, also, `4.0.0` already released.

Reply:
yah this PR is older than v4.0.0 release. Will update at some point.